pkg/didcomm/packer/jwe/authcrypt/unpack.go

```diff
@@ -47,18 +47,18 @@ func (p *Packer) Unpack(envelope []byte) ([]byte, error) {
 
 		sharedKey, er := p.decryptCEK(recipientPubKey, senderPubKey, recipient)
 		if er != nil {
-			return nil, fmt.Errorf("unpack: decrypt shared key: %w", er)
+			return nil, nil, fmt.Errorf("unpack: decrypt shared key: %w", er)
 		}
 
 		symOutput, er := p.decryptPayload(sharedKey, jwe)
 		if er != nil {
-			return nil, fmt.Errorf("unpack: %w", er)
+			return nil, nil, fmt.Errorf("unpack: %w", er)
 		}
 
-		return symOutput, nil
+		return symOutput, senderPubKey[:], nil
```

Comment:
You could return `senderKey` instead but it makes no difference here.

Reply:
(It's simpler, but it's fine this way)

Reply:
Valid point (missed this during refactoring). [Updated](https://github.com/hyperledger/aries-framework-go/compare/91457d1236b61875c4b0ce2b529d75fe2851dbc3..b34c5cd85fb496c143d5c08f96451194c349a7c8#diff-798c9657f10f55743907e471a65aee97R58) the code to use `senderKey`.